src/index.scss

```diff
@@ -89,4 +89,18 @@ button {
     border: 0;
     padding: 0;
   }
-}
\ No newline at end of file
+}
+
+/* Sticky table headers */
+thead th {
```

Comment:
This can't be made global. It'll screw up the chart labels. We need a class name for sticky header.